docs/build/manifest.md

```diff
@@ -5,39 +5,86 @@ The Manifest `project.yaml` file can be seen as an entry point of your project a
 The Manifest can be in either YAML or JSON format. In this document, we will use YAML in all the examples. Below is a standard example of a basic `project.yaml`.
 
 <CodeGroup>
+
+
+<CodeGroupItem title="v1.0.0" active>
+
+``` yml
+
+specVersion: 1.0.0
+name: subquery-starter
+version: 0.0.4
+runner:
+  node:
+    name: '@subql/node'
```

Comment:
Since we name avalanche as `@subql/node-avalanche` can we add an alias for `@subql/node-substrate`

Reply:
I think we can remain `@subql/node` in the document for now, in the next release 1.0.0 for node, we can change it.

Reply:
@stwiname @ianhe8x what you guys think on update the node name

Reply:
I think we should keep `@subql/node` but can also publish to `@subql/node-substrate`. I don't think npm has any sort of alias functionality